byte-streams-explainer.md

```diff
@@ -86,3 +102,23 @@ function makeReadableByteStream() {
 With this in hand, we can create and use BYOB readers for the returned `ReadableStream`. The adaptation between the
 low-level byte tracking of the underlying byte source shown here, and the higher-level chunk-based consumption of
 a default reader, is all taken care of automatically by the streams implementation.
+
+
+## Goals
+
+*   Provide a way to represent a [ReadableStream](https://streams.spec.whatwg.org/#rs-class) for bytes efficiently.
+*   Avoid races caused by multiple access for the same buffer.
+
+## Non-Goals
+
+*   Ports for non-binary types will not be supported; only Bytes, Int16, etc. are supported by the default readable
```

Comment:
I find this a little confusing, since the default readable stream supports all types. How about something like

> Non-binary chunk types will not be supported. They can still use the default type of readable stream.

?

Reply:
I agree, that makes more sense. Thank you!